addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -232,6 +232,20 @@ var Wysiwyg = Widget.extend({
                             },
                         },
                     },
+
+                    // js_language_selector
+                    {
+                        selector: [node => {
+                            const attributes = node.modifiers.find(this.JWEditorLib.Attributes);
+                            return attributes && attributes.classList.has('js_language_selector');
```

Comment:
Hello, isn't it weird to have website related data in `web_editor` ?

Reply:
Yes right now the rules are set in one place only, it will require a later commit to allow it to be split so we can refactor this. Just hasn't been our top priority but it is in the cards ;-)